install.md

```diff
@@ -1,5 +1,27 @@
 # libpod Installation Instructions
 
+## Installing packaged versions of Podman
+
+#### Fedora, Centos, OpenSUSE, SUSE:
+
+yum -y install podman
+
+#### RHEL7
+
+Enable the extras repo.
```

Comment:
is there a quick way to do that, or all tied up with subscription fun?

Reply:
Subscribe, then enable Extras channel and install podman.

```bash
$ subscription-manager repos --enable=rhel-7-server-extras-rpms
$ sudo yum -y install podman
```